src/shared/components/PageHeader/PageHeader.tsx

```diff
@@ -29,15 +30,26 @@ const PageTitle = styled.h1`
 interface PageHeaderProps {
   isOpen: boolean;
   openSideBar: () => void;
-  selectedMenu: string;
 }
 
-function PageHeader({ isOpen, openSideBar, selectedMenu }: PageHeaderProps) {
+function PageHeader({ isOpen, openSideBar }: PageHeaderProps) {
+  const location = useLocation();
+  const path = location.pathname;
+
+  let title = '페이지';
+
+  if (path.startsWith('/dashboard')) title = '대시보드';
+  else if (path.startsWith('/create')) title = '문제집 생성';
+  else if (path.startsWith('/solve')) title = '문제 풀이';
+  else if (path.startsWith('/library')) title = '나의 문제집';
+  else if (path.startsWith('/wrong')) title = '오답노트';
+  else if (path === '/') title = '문제집 생성';
```

Comment:
path가 바뀔때만 계산하도록 useMemo로 감싸면 좋을거 같습니다!
```suggestion
const title = useMemo(() => {
  if (path.startsWith('/dashboard')) return '대시보드';
  if (path.startsWith('/create')) return '문제집 생성';
  if (path.startsWith('/solve')) return '문제 풀이';
  if (path.startsWith('/library')) return '나의 문제집';
  if (path.startsWith('/wrong')) return '오답노트';
  if (path === '/') return '문제집 생성';
  return '페이지';
}, [path]);
```

Reply:
최종 결과물에 반영하겠습니다!